### PR TITLE
Multiple accounts to customize support levels

### PR DESCRIPTION
**Pull request by tomjanofsky**
Add multiple accounts description to note that you can split your support bill this way as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ Billing and Cost Management
...
 -	If you need to do custom analysis of raw billing data or want to feed it to a third party cost analysis service, [enable](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/detailed-billing-reports.html#turnonreports) the [detailed billing report](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/detailed-billing-reports.html#detailed-billing-report) feature.
--	Multiple Amazon accounts can be linked for billing purposes using the [Consolidated Billing](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/consolidated-billing.html) feature. Large enterprises may need complex billing structures depending on ownership and approval processes.
+-	Multiple Amazon accounts can be linked for billing purposes using the [Consolidated Billing](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/consolidated-billing.html) feature. Large enterprises may need complex billing structures depending on ownership and approval processes. Multiple accounts can also be used to deploy different parts of your infrastrcuture under different support levels. You can use this to tune your support level to your desired spend.
```
Comment (jlevy) [inline README.md]: This is a great tip! Actually, could you make it a separate bullet below this, with a 🔹 in front so it's highlighted?

**Comment by jlevy** [comment]
The tip is a great addition. It looks like there was a bad merge on the table change. Can you update or make a PR with just the single tip change?

File: README.md (hunk 1 above)
Comment (max-zanko) [inline README.md]: This comes with strings attached. Multiple accounts:
a) security and access configuration [slightly] more complicated and
b) potentially push your into the lower volume, higher price usage tiers.
@jlevy, should we mention these here?

Also, s/infrastrcuture/infrastructure/

**Comment by QuinnyPig** [comment]
Closing for now as @tomjanofsky has gone quiet-- please don't hesitate to reopen / resolve the merge conflicts. This is good stuff!

**Pull request closed without merging**